Yoast/wordpress-seo
# Remove post-type-archive-notification

## Context


* Removes post-type-archive-notification.

## Summary


This PR can be summarized in the following changelog entry:

* Removes post-type-archive-notification.

## Relevant technical choices:

* I've used version `14.1` for the `@deprecated` and upgrade routine.
* Completely removes test file for `WPSEO_Post_Type_Archive_Notification_Handler`.

## Test instructions

This PR can be tested by following these steps:

Please look carefully whether the deprecating was done correctly, thanks :).

* _In trunk_: build 
* In the database: `<wp-prefix>-options` > `wpseo` > change `"first_activated_on";i:1580984679” `(probably different value) to `"first_activated_on";i:1532383200” `. Save the old value to put it back later.
* alternatively, run this code: `WPSEO_Options::set( 'first_activated_on', 1532383200 );`
* activate the Yoast Test Helper plugin and activate the "Book" custom post type
* In the backend, go to `SEO` > `General` and see the following notification
<img width="876" alt="post-type-archive-notification" src="https://user-images.githubusercontent.com/43582255/79452120-ee469f00-7fe7-11ea-97f3-f44aace6541d.png">

* Go to this branch and build
* Trigger the upgrade routine (I didn't get it to work with the Yoast test helper, but reactivating the plugin did work)
* Reload `SEO` > `General` and see that the notification is not there

## UI changes

## Documentation

## Quality assurance


Fixes #14496
